### PR TITLE
Add review process page + updated issue templates

### PR DESCRIPTION
**Pull request by nabobalis**
I also redid the labels.

I added an untested assignment actions to open issues.

### CODE REVIEW

File: .github/workflows/assigment.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  auto-assign:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: "Auto-assign issue"
+        uses: pozil/auto-assign-issue@v1
+        with:
+          teams: instrastructure
+          repo-token: ${{ secrets.infra_automation }}
```
Comment (nabobalis) [inline .github/workflows/assigment.yml:17]: This should hopefully work.

File: docs/review_process.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  - [ ] Request topic is covered by the infrastructure committee remit.
+
+  - [ ] If the request requires expenditure, the price is within the budget of the committee.
+
+  ## Review Comments (If any)
+
+Appeal Process
+--------------
+
+In the cases where project members feel that their issue was unfairly closed, there will be no appeals process.
```
Comment (aterrel) [inline docs/review_process.rst:45]: Can we just remove this section if there is no appeals process?

File: docs/review_process.rst [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+Reporting to the Board
+----------------------
+
+Updates on all project requests that were denied will be given to the board to ensure that the infrastructure committee is not rejecting valid requests.
```
Comment (nabobalis) [inline docs/review_process.rst:50]:
```suggestion
The infrastructure committee provides quarterly reports of our requests and results.
The committee strives to fulfill every request made, but in the event that a request is denied due to lack of resources, we can petition the board to fund more resources.
```

File: docs/review_process.rst [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+  - [ ] If the request requires expenditure, the price is within the budget of the committee.
+
+  ## Review Comments (If any)
+
+Appeal Process
+--------------
+
+In the cases where project members feel that their issue was unfairly closed, there will be no appeals process.
+
```
Comment (nabobalis) [inline docs/review_process.rst:46]:
```suggestion
```

**Pull request merged**